index.rst

```diff
@@ -2,24 +2,54 @@
 
 .. sectnum::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+Abstract
+========
 
-.. TODO: Delete the note below before merging new content to the main branch.
+This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
 
-.. note::
+Description of the Incident
+===========================
 
-   **This technote is a work-in-progress.**
+This incident initiated between 8:00-8:30 CLT/3:00-3:30PST December, 6 2022, during the first night of the 2022A run on the Auxiliary Telescope.
+We experienced a catastrophic network failure that took down the network in the entire observatory.
+The failure extended to both the internal and external network, both over the WiFi and the cable connections.
+All WiFi connections were dropped and we could no longer connect to the network.
 
-Abstract
-========
+Up until the issue started we were preparing to go on sky with the Auxiliary Telescope.
+The TMA was experiencing power issues and would not be able to operate during the night for the soak and start tracker testing. 
+Alysha Shugart and Tiago Ribeiro were executing some dome movement tests with the remote help of Bruno Quint.
 
-This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
+At one point around 8:30 PM local time, Tiago Ribeiro went to update LOVE in the love02 node with the latest version of the logging system for testing during the night.
+The node was completely shutdown (no LOVE system or ospl daemon running), shortly after the ospl daemon on love2 was initialized the network outage started.
+We were able to communicate with La Serena over Slack using a poor cellular connection and after being able to raise IT support it was established that it was impossible to recover from the issue remotely.
+
+At that point the crew decided to call in the night.
+The Auxiliary Telescope dome was manually closed and the telescope was left in the position it was in.
+The mirror covers were closed and all equipment was left in a safe state.
+
+The next morning IT arrived at the summit and determined that the issue was caused because two hosts were broadcasting using the same IP, which created an internal loop, causing the ACI to completely shutdown the network.
```

Comment:
I don't suppose we can get IT's input on how the ACI system contributed to this?